### PR TITLE
Scroll messages and other lang fixes

### PR DESCRIPTION
**Pull request by ravenbuilder934**
Closes #
Closes #
Closes #

# Changes proposed in this pull request:
- Supersedes #6173, but without the merge conflicts and extra messages!
- *Just grammar Sam, don't freak out*
- Also has some other lang fixes and dead code removal

Review please

### CODE REVIEW

File: src/main/resources/assets/minecolonies/lang/en_us.json [hunk 1]
```diff
@@ -363,31 +363,31 @@
...
-  "item.minecolonies.scroll_area_tp": "Ultrasafe colony group-teleport scroll",
-  "item.minecolonies.scroll_area_tp.tip": "Teleport yourself and nearby players to the colony",
-  "item.minecolonies.scroll_buff": "Sacred Scroll of Regeneration",
+  "item.minecolonies.scroll.colony.none": "None, set one with sneak + use",
+  "item.minecolonies.scroll_area_tp": "Colony Group Portal Scroll",
+  "item.minecolonies.scroll_area_tp.tip": "Teleports you and nearby players to the colony",
+  "item.minecolonies.scroll_buff": "Regeneration Scroll",
   "item.minecolonies.scroll_buff.tip": "Applies regeneration to citizens around you, at least most of the time...",
-  "item.minecolonies.scroll_guard_help": "Spatial guard reinforcement scroll",
-  "item.minecolonies.scroll_guard_help.tip": "Call your guards to help you out for a limited time, wololo!",
```
Comment (Mekle001) [inline src/main/resources/assets/minecolonies/lang/en_us.json:360]: Ok, I think I get why you are removing the wololo, but it makes me sad to lose the reference. This was one of the more funny recent additions.

Comment (ravenbuilder934) [reply src/main/resources/assets/minecolonies/lang/en_us.json:360]: Yeah, sadly it doesn't really fit as that's not what wololo actually did

Comment (Mekle001) [reply src/main/resources/assets/minecolonies/lang/en_us.json:360]: Perhaps find a way to work it into the regeneration scroll?

Comment (ravenbuilder934) [reply src/main/resources/assets/minecolonies/lang/en_us.json:360]: It doesn't regenerate either, it changes enemies to allies. Maybe if we add extra things we can do to raiders (like turning them into citizens) we can work it into there?

Comment (Mekle001) [reply src/main/resources/assets/minecolonies/lang/en_us.json:360]: Fair, though it invokes the AoE priest which did both conversion and healing, so regeneration would be the closest reference we have today.

**Review by Mekle001 — DISMISSED** [review]

**Comment by someaddons** [comment]
No

**Comment by ravenbuilder934** [comment]
Why not? This is just my normal grammar/etc fixes, no new messages.

**Comment by someaddons** [comment]
its not just grammar, you're mainly changing names and meaning of many messages :D

**Comment by ravenbuilder934** [comment]
Okay, what if I did this @someaddons?
```json
  "item.minecolonies.scroll_tp": "Colony Teleport Scroll",

  "item.minecolonies.scroll_tp.tip": "Teleports you to the set colony",

  "item.minecolonies.scroll.colony.tip": "Colony: %s",

  "item.minecolonies.scroll.colony.none": "None, set one with sneak + use",

  "item.minecolonies.scroll_area_tp": "Colony Group Teleport Scroll",

  "item.minecolonies.scroll_area_tp.tip": "Teleports you and nearby players to the colony",

  "item.minecolonies.scroll_buff": "Regeneration Scroll",

  "item.minecolonies.scroll_buff.tip": "Applies regeneration to citizens around you, at least most of the time...",

  "item.minecolonies.scroll_guard_help": "Reinforcements Scroll",

  "item.minecolonies.scroll_guard_help.tip": "Call your guards to help you out for a limited time",

  "item.minecolonies.scroll_highlight": "Worker Locator Scroll",

  "item.minecolonies.scroll_highlight.tip": "Sneak + use on a hut block to highlight and speed up its worker(s)",

  "minecolonies.scroll.nopermission": "You do not have the required permissions for this colony.",

  "minecolonies.scroll.needcolony": "Please register this scroll to a colony first by sneaking + using it on a hut block.",

  "minecolonies.scroll.noguardbuilding": "Please register this scroll to a Guard or Barracks Tower first by sneaking + using it on the hut block.",

  "minecolonies.scroll.wrongdim": "This scroll can only be used in the same dimension as the colony.",

  "minecolonies.scroll.registered": "Registered scroll with colony %s.",

  "minecolonies.scroll.failed1": "The unstable magic of the scroll backfired!",

  "minecolonies.scroll.failed2": "Whoops, what happened?!",

  "minecolonies.scroll.failed3": "This scroll needs to go back to the drawing board!",

  "minecolonies.scroll.failed4": "This scroll doesn't seem to be made by the best magician out there!",

  "minecolonies.scroll.failed5": "Well, this is embarrassing.",

  "minecolonies.scroll.failed6": "Looks like there's a typo in one of the ancient glyphs!",

  "minecolonies.scroll.failed7": "Did my creator copy the wrong rune again?!",

  "minecolonies.scroll.failed8": "Wasn't this supposed to do something?",

  "minecolonies.scroll.failed9": "404 Spell Not Found",

  "minecolonies.scroll.failed10": "Maybe try some ladders instead?",`
```

File: src/main/resources/assets/minecolonies/lang/en_us.json [hunk 2]
```diff
@@ -345,30 +345,30 @@
...
+  "item.minecolonies.scroll_guard_help": "Spatial Guard Reinforcements Scroll",
+  "item.minecolonies.scroll_guard_help.tip": "Calls your Guards to help you out for a limited time",
+  "item.minecolonies.scroll_highlight": "Worker-Where-Are-You Scroll",
+  "item.minecolonies.scroll_highlight.tip": "Sneak + right-click on a MineColonies building to highlight and speed up its worker(s)",
   "minecolonies.scroll.nopermission": "You do not have the required permissions for this colony.",
-  "minecolonies.scroll.needcolony": "Please register this to a colony first with shift + right-clicking a building.",
-  "minecolonies.scroll.noguardbuilding": "Please register this to a guard building by shift + right-clicking it.",
-  "minecolonies.scroll.wrongdim": "Can only be used in the same dimension as the colony.",
-  "minecolonies.scroll.registered": "Registered scroll with colony %s",
+  "minecolonies.scroll.needcolony": "Please register this scroll to a colony first by sneaking + right-clicking it on a MineColonies building.",
```
Comment (Raycoms) [inline src/main/resources/assets/minecolonies/lang/en_us.json]: Should probably avoid calling it "Minecolonies" building for immersion but just a "colony building"

**Review by someaddons — APPROVED** [review]

**Pull request merged**